Assets/Mirror/Examples/Room/Scripts/NetworkRoomPlayerExt.cs

```diff
@@ -6,11 +6,14 @@ namespace Mirror.Examples.NetworkRoom
     [AddComponentMenu("")]
     public class NetworkRoomPlayerExt : NetworkRoomPlayer
     {
-        public override void OnStartClient()
+        void Awake()
         {
-            if (LogFilter.Debug) Debug.LogFormat("OnStartClient {0}", SceneManager.GetActiveScene().name);
+            netIdentity.OnStartClient.AddListener(OnStartClient);
+        }
 
-            base.OnStartClient();
+        public void OnStartClient()
```

Comment:
Not doing anything,  maybe just get rid of it?

Reply:
It provides logging on the next line. Do you want the logging removed too?